### DISCUSSION TITLE
Consulta de los tests.

### DISCUSSION THREAD

**Discussion by eayaach** [IIC2133-PUC/2024-2#106] [Tarea 1] [answered]
En el enunciado se menciona que para tener todo el puntaje se debe cumplir con cada test en menos de 5 segundos. Al ejecutar mi codigo se ejecuta en mucho menos de 5 segundos sin usar valgrind time pero con time se demora el triple. ¿ debo guiarme por lo de valgrind o con la ejecución normal?

**Comment by tomastrivino** [comment, marked as answer]
Hola,
guíate por la ejecución normal (con el comando `time`). ¡Valgrind aumenta considerablemente el tiempo!
